docs/UserGuide.adoc

```diff
@@ -133,6 +133,21 @@ Views all details of the 2nd person in the address book.
 `viewall 1` +
 Views all details of the 1st person in the results of the `find` command.
 
+
+== Sort the address book : `sort`
+
+Sorts the entire address book according to lexicographical/colexicographical order. +
+Format: `sort [ascending | asc] | [descending | desc]`
+
```

Comment:
Small small nitpick: '|' meaning is not specified
With respect to the original UG under add command, 

**"Words in `UPPER_CASE` are the parameters, items in `SQUARE_BRACKETS` are optional".** 

They described what the notation is before adopting it in the UG, so you should do so too